src/views/SheSaidYes.js

```diff
@@ -10,35 +10,17 @@ import Span from "../components/Span";
 const SheSaidYes = ({ view, invert }) => (
   <section>
     <Article left view={view} wide={invert}>
-      <Heading top="10em" left="7%" align="left">
+      <Heading
+        className="heading--with-graphic"
```

Comment:
Alternative:

Instead of a `className`, a prop can be passed to the `<Heading />` component.

```jsx
<Heading withGraphic ... />
```

In `src/components/Heading.js`...

```javascript
const StyledHeading = styled.h2`
  position: absolute;
  font-size: ${props => (props.size ? props.size : "1rem")};
  font-weight: bold;
  color: #868fa4;
  opacity: 0.4;
  width: ${props => (props.width ? props.width : "100%")};
  top: ${props => (props.top ? props.top : "auto")};
  bottom: ${props => (props.bottom ? props.bottom : "auto")};
  left: ${props => (props.left ? props.left : "auto")};
  right: ${props => (props.right ? props.right : "auto")};
  text-align: ${props => (props.align ? props.align : "left")};

  ${props => props.withGraphic && `
    &::after {
      position: absolute;
      content: "";
      background-image: url("/img/graphics/heading_hearts.svg");
      background-repeat: no-repeat;
      right: 5em;
      top: -1em;
      height: 12em;
      width: 8.625em;
    }
  `}
`;
```